### ISSUE TITLE
Bootstrap checks fail when starting a single-node cluster via Docker as suggested by the official tutorial

### ISSUE THREAD

**Issue by thalesfm** [elastic/elasticsearch#91773]
### Elasticsearch Version

8.5.1

### Java Version

_bundled_

### OS Version

Fedora 37

### Problem Description

By following the [official tutorial](https://www.elastic.co/guide/en/elasticsearch/reference/8.5/docker.html) on installing Elasticsearch via Docker for version 8.5.1, step 2 of the "Start a single node cluster with Docker" section produces an error due to failing bootstrap checks. Enabling single-node discovery (as [suggested by the tutorial for version 7.17](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docker.html#docker-cli-run-dev-mode)) seems to fix the issue (though I suspect this is because in this case these same checks are bypassed).

### Steps to Reproduce

On a fresh Fedora 37 installation, just follow the exact steps described in the tutorial to reproduce the issue:

```
$ docker pull docker.elastic.co/elasticsearch/elasticsearch:8.5.1
$ docker network create elastic
$ docker run --name es01 --net elastic -p 9200:9200 -p 9300:9300 -it docker.elastic.co/elasticsearch/elasticsearch:8.5.1
```

Enabling single-node discovery seems to fix it:

```
docker run --name es01 --net elastic -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -it docker.elastic.co/elasticsearch/elasticsearch:8.5.1
```

### Logs (if relevant)

{"@timestamp":"2022-11-21T13:53:18.091Z", "log.level": "INFO", "message":"version[8.5.1], pid[73], build[docker/c1310c45fc534583afe2c1c03046491efba2bba2/2022-11-09T21:02:20.169855900Z], OS[Linux/6.0.8-300.fc37.x86_64/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/19.0.1/19.0.1+10-21]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:18.099Z", "log.level": "INFO", "message":"JVM home [/usr/share/elasticsearch/jdk], using bundled JDK [true]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:18.099Z", "log.level": "INFO", "message":"JVM arguments [-Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -Djava.security.manager=allow, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Dlog4j2.formatMsgNoLookups=true, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.io=ALL-UNNAMED, -Des.cgroups.hierarchy.override=/, -XX:+UseG1GC, -Djava.io.tmpdir=/tmp/elasticsearch-8074086217683437686, -XX:+HeapDumpOnOutOfMemoryError, -XX:+ExitOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Xms7939m, -Xmx7939m, -XX:MaxDirectMemorySize=4162846720, -XX:G1HeapRegionSize=4m, -XX:InitiatingHeapOccupancyPercent=30, -XX:G1ReservePercent=15, -Des.distribution.type=docker, --module-path=/usr/share/elasticsearch/lib, --add-modules=jdk.net, -Djdk.module.main=org.elasticsearch.server]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:19.818Z", "log.level": "INFO", "message":"Package versions: jackson-annotations=2.13.2, jackson-core=2.13.2, jackson-databind=2.13.2.2, jackson-dataformat-xml=2.13.2, jackson-datatype-jsr310=2.13.2, azure-core=1.27.0, Troubleshooting version conflicts: https://aka.ms/azsdk/java/dependency/troubleshoot", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"com.azure.core.implementation.jackson.JacksonVersion","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.047Z", "log.level": "INFO", "message":"loaded module [aggs-matrix-stats]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.047Z", "log.level": "INFO", "message":"loaded module [analysis-common]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.048Z", "log.level": "INFO", "message":"loaded module [apm]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.048Z", "log.level": "INFO", "message":"loaded module [constant-keyword]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.048Z", "log.level": "INFO", "message":"loaded module [data-streams]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.048Z", "log.level": "INFO", "message":"loaded module [frozen-indices]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.048Z", "log.level": "INFO", "message":"loaded module [ingest-attachment]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.048Z", "log.level": "INFO", "message":"loaded module [ingest-common]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [ingest-geoip]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [ingest-user-agent]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [kibana]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [lang-expression]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [lang-mustache]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [lang-painless]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [legacy-geo]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.049Z", "log.level": "INFO", "message":"loaded module [mapper-extras]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [mapper-version]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [old-lucene-versions]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [parent-join]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [percolator]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [rank-eval]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [reindex]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [repositories-metering-api]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [repository-azure]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.050Z", "log.level": "INFO", "message":"loaded module [repository-encrypted]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [repository-gcs]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [repository-s3]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [repository-url]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [runtime-fields-common]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [search-business-rules]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [searchable-snapshots]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [snapshot-based-recoveries]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.051Z", "log.level": "INFO", "message":"loaded module [snapshot-repo-test-kit]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [spatial]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [transform]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [transport-netty4]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [unsigned-long]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [vector-tile]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [wildcard]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [x-pack-aggregate-metric]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [x-pack-analytics]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.052Z", "log.level": "INFO", "message":"loaded module [x-pack-async]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-async-search]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-autoscaling]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-ccr]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-core]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-deprecation]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-enrich]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-eql]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-fleet]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.053Z", "log.level": "INFO", "message":"loaded module [x-pack-graph]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-identity-provider]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-ilm]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-logstash]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-ml]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-monitoring]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-ql]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-rollup]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-security]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.054Z", "log.level": "INFO", "message":"loaded module [x-pack-shutdown]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.055Z", "log.level": "INFO", "message":"loaded module [x-pack-sql]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.055Z", "log.level": "INFO", "message":"loaded module [x-pack-stack]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.055Z", "log.level": "INFO", "message":"loaded module [x-pack-text-structure]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.055Z", "log.level": "INFO", "message":"loaded module [x-pack-voting-only-node]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.055Z", "log.level": "INFO", "message":"loaded module [x-pack-watcher]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:21.055Z", "log.level": "INFO", "message":"no plugins loaded", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:23.833Z", "log.level": "WARN", "message":"Nov 21, 2022 1:53:23 PM org.apache.lucene.store.MMapDirectory lookupProvider", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:23.833Z", "log.level": "WARN", "message":"WARNING: You are running with Java 19. To make full use of MMapDirectory, please pass '--enable-preview' to the Java command line.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:23.842Z", "log.level": "INFO", "message":"using [1] data paths, mounts [[/ (/dev/sdc3)]], net usable_space [99gb], net total_space [110.2gb], types [btrfs]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.env.NodeEnvironment","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:23.842Z", "log.level": "INFO", "message":"heap size [7.7gb], compressed ordinary object pointers [true]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.env.NodeEnvironment","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:23.855Z", "log.level": "INFO", "message":"node name [4959fbd1ac84], node ID [oQ8dysSrRSuo4JOpL7t6PQ], cluster name [docker-cluster], roles [ingest, data_frozen, ml, data_hot, transform, data_content, data_warm, master, remote_cluster_client, data, data_cold]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:26.896Z", "log.level": "INFO", "message":"Security is enabled", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.xpack.security.Security","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:27.272Z", "log.level": "INFO", "message":"parsed [0] roles from file [/usr/share/elasticsearch/config/roles.yml]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.xpack.security.authz.store.FileRolesStore","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}

{"@timestamp":"2022-11-21T13:53:27.708Z", "log.level": "INFO", "message":"[controller/97] [Main.cc@123] controller (64 bit): Version 8.5.1 (Build ac5909e3891a6b) Copyright (c) 2022 Elasticsearch BV", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"ml-cpp-log-tail-thread","log.logger":"org.elasticsearch.xpack.ml.process.logging.CppLogMessageHandler","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:28.176Z", "log.level": "INFO", "message":"creating NettyAllocator with the following configs: [name=elasticsearch_configured, chunk_size=1mb, suggested_max_allocation_size=1mb, factors={es.unsafe.use_netty_default_chunk_and_page_size=false, g1gc_enabled=true, g1gc_region_size=4mb}]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.transport.netty4.NettyAllocator","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:28.197Z", "log.level": "INFO", "message":"using rate limit [40mb] with [default=40mb, read=0b, write=0b, max=0b]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.indices.recovery.RecoverySettings","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:28.229Z", "log.level": "INFO", "message":"using discovery type [multi-node] and seed hosts providers [settings]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.discovery.DiscoveryModule","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.373Z", "log.level": "INFO", "message":"initialized", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.374Z", "log.level": "INFO", "message":"starting ...", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.393Z", "log.level": "INFO", "message":"persistent cache index loaded", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.xpack.searchablesnapshots.cache.full.PersistentCache","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.394Z", "log.level": "INFO", "message":"deprecation component started", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.xpack.deprecation.logging.DeprecationIndexingComponent","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.473Z", "log.level": "INFO", "message":"publish_address {172.18.0.2:9300}, bound_addresses {0.0.0.0:9300}", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.transport.TransportService","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.580Z", "log.level": "INFO", "message":"bound or publishing to a non-loopback address, enforcing bootstrap checks", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.bootstrap.BootstrapChecks","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
bootstrap check failure [1] of [1]: max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
ERROR: Elasticsearch did not exit normally - check the logs at /usr/share/elasticsearch/logs/docker-cluster.log
{"@timestamp":"2022-11-21T13:53:29.588Z", "log.level": "INFO", "message":"stopping ...", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"Thread-0","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.635Z", "log.level": "INFO", "message":"stopped", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"Thread-0","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.635Z", "log.level": "INFO", "message":"closing ...", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"Thread-0","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.645Z", "log.level": "INFO", "message":"closed", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"Thread-0","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2022-11-21T13:53:29.646Z", "log.level": "INFO", "message":"Native controller process has stopped - no new native processes can be started", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"ml-cpp-log-tail-thread","log.logger":"org.elasticsearch.xpack.ml.process.NativeController","elasticsearch.node.name":"4959fbd1ac84","elasticsearch.cluster.name":"docker-cluster"}
2022-11-21 13:53:29,652 process reaper (pid 97) ERROR Recursive call to appender rolling
{"@timestamp":"2022-11-21T13:53:29.647Z", "log.level":"ERROR", "message":"uncaught exception in thread [process reaper (pid 97)]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"process reaper (pid 97)","log.logger":"org.elasticsearch.bootstrap.ElasticsearchUncaughtExceptionHandler","elasticsearch.node.name":"4959fbd1ac84","error.type":"java.security.AccessControlException","error.message":"access denied (\"java.lang.RuntimePermission\" \"modifyThread\")","error.stack_trace":"java.security.AccessControlException: access denied (\"java.lang.RuntimePermission\" \"modifyThread\")\n\tat java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:485)\n\tat java.base/java.security.AccessController.checkPermission(AccessController.java:1068)\n\tat java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:411)\n\tat org.elasticsearch.securesm@8.5.1/org.elasticsearch.secure_sm.SecureSM.checkThreadAccess(SecureSM.java:166)\n\tat org.elasticsearch.securesm@8.5.1/org.elasticsearch.secure_sm.SecureSM.checkAccess(SecureSM.java:120)\n\tat java.base/java.lang.Thread.checkAccess(Thread.java:2360)\n\tat java.base/java.lang.Thread.setDaemon(Thread.java:2308)\n\tat java.base/java.lang.ProcessHandleImpl.lambda$static$0(ProcessHandleImpl.java:103)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.<init>(ThreadPoolExecutor.java:637)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.addWorker(ThreadPoolExecutor.java:928)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.processWorkerExit(ThreadPoolExecutor.java:1021)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1158)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1589)\n\tat java.base/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:186)\n"}

ERROR: [1] bootstrap checks failed. You must address the points described in the following [1] lines before starting Elasticsearch.

**Comment by rjernst** [comment]
Duplicate of https://github.com/elastic/elasticsearch/issues/91658, which is now consolidated in https://github.com/elastic/elasticsearch/issues/91650.
